### PR TITLE
feat: Support for update custom reward redemption status

### PR DESCRIPTION
**Pull request by SaxyPandaBear**
Thankful for the project, but it's missing this API that I need. I don't really know how to test it locally e2e so if you could explain that I can try it out. Otherwise, here you go. I just tried my best to follow the spec https://dev.twitch.tv/docs/api/reference/#update-redemption-status and the existing code/tests. Ran `go test ./... -cover` and it all passed.

### CODE REVIEW

File: channels_points.go [hunk 1]
```diff
@@ -104,6 +106,36 @@ type DeleteCustomRewardsResponse struct {
...
+	ID               string              `json:"id"`
+	BroadcasterID    string              `json:"broadcaster_id"`
+	BroadcasterLogin string              `json:"broadcaster_login"`
+	BroadcasterName  string              `json:"broadcaster_name"`
+	UserID           string              `json:"user_id"`
+	UserName         string              `json:"user_name"`
+	UserLogin        string              `json:"user_login"`
+	UserInput        string              `json:"user_input"`
+	Status           string              `json:"status"`
+	RedeemedAt       time.Time           `json:"redeemed_at"`
```
Comment (nicklaw5) [inline channels_points.go:135]: This package has it's own implementation for handling [time](https://github.com/nicklaw5/helix/blob/main/time.go) values. Please make the following adjustment:

```suggestion
	RedeemedAt       Time           `json:"redeemed_at"`
```

File: channels_points.go [hunk 2]
```diff
@@ -1,5 +1,7 @@
 package helix
 
+import "time"
```
Comment (nicklaw5) [inline channels_points.go:3]: This package can be removed also.

**Review by nicklaw5 — CHANGES_REQUESTED** [review]
Thanks for the contribution. Just the one comment regarding `Time`.

>  I don't really know how to test it locally e2e so if you could explain that I can try it out.

This needs to be done with your own API token. We use mocking in unit tests to simulate 
Twitch requests and responses, but there is no e2e testing.

File: channels_points.go (hunk 1 above)
Comment (SaxyPandaBear) [reply channels_points.go]: Updated. Sorry I guess I missed that while I was skimming the package

**Review by nicklaw5 — APPROVED** [review]

**Pull request merged**